src/app/profile/cleanup/cleanup.component.spec.ts

```diff
@@ -51,7 +72,7 @@ describe('CleanupComponent', () => {
     fixture = TestBed.createComponent(CleanupComponent);
     component = fixture.debugElement.componentInstance;
     component.confirmCleanup = jasmine.createSpyObj('IModalHost', ['open', 'close']);
-    mockSpace = jasmine.createSpy('Space');
+    component.ngOnInit();
```

Comment:
`fixture.detectChanges()` should trigger the component lifecycle hooks, and IMO is preferable to manually calling the hooks like this.

Reply:
Ah that's neat, I'll make this and re-push in a sec